test/integration/test_huobi_market_mock.py

```diff
@@ -1,472 +0,0 @@
-#!/usr/bin/env python
-from os.path import (
-    join,
-    realpath
-)
-import sys; sys.path.insert(0, realpath(join(__file__, "../../../")))
-
-from aiohttp import web
-from aiohttp.test_utils import (
-    AioHTTPTestCase,
-    TestClient
-)
-import asyncio
-import contextlib
-from decimal import Decimal
-import logging
-import os
-import time
-from typing import (
-    List,
-    Optional
-)
-import unittest
-
-from hummingbot.core.clock import (
-    Clock,
-    ClockMode
-)
-from hummingbot.core.event.event_logger import EventLogger
-from hummingbot.core.event.events import (
-    MarketEvent,
-    BuyOrderCompletedEvent,
-    SellOrderCompletedEvent,
-    OrderFilledEvent,
-    OrderCancelledEvent,
-    BuyOrderCreatedEvent,
-    SellOrderCreatedEvent,
-    TradeFee,
-    TradeType,
-)
-from hummingbot.core.utils.async_utils import (
-    safe_ensure_future,
-    safe_gather,
-)
-from hummingbot.logger.struct_logger import METRICS_LOG_LEVEL
-from hummingbot.market.huobi.huobi_market import HuobiMarket
-from hummingbot.market.huobi.huobi_order_book import HuobiOrderBook
-from hummingbot.market.market_base import OrderType
-from hummingbot.market.markets_recorder import MarketsRecorder
-from hummingbot.market.mock_api_order_book_data_source import MockAPIOrderBookDataSource
-from hummingbot.model.market_state import MarketState
-from hummingbot.model.order import Order
-from hummingbot.model.sql_connection_manager import (
-    SQLConnectionManager,
-    SQLConnectionType
-)
-from hummingbot.model.trade_fill import TradeFill
-from test.integration.huobi_mock_api import HuobiMockAPI
-
-logging.basicConfig(level=METRICS_LOG_LEVEL)
-MOCK_HUOBI_API_KEY = "zzzzzzzzzz-xxxxxxxx-55555555-xxxxx"
-MOCK_HUOBI_SECRET_KEY = "11111111-aaaaaaaa-22222222-bbbbb"
-
-
-class HuobiMarketUnitTest(AioHTTPTestCase):
```

Comment:
Why is the entire Huobi Market test removed?

Reply:
This is an old mock test, we don't need this anymore since the mock test mode is now embeded in the same test file.